react/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { func } from 'prop-types';
+
+import styles from './styles.module.scss';
+
+const validate = values => {
+  const errors = {};
+  if (!values.email) {
+    errors.email = 'Por favor ingresa un email';
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+    errors.email = 'Selecciona una dirección de email válido';
+  }
+
+  if (!values.password) {
+    errors.password = 'Selecciona una contraseña';
+  } else if (values.password.length < 8) {
+    errors.password = 'Debe tener al menos 8 caracteres';
+  }
+
+  return errors;
+};
+
+class Login extends Component {
+  renderField = ({ input, label, type, placeholder, meta: { touched, error } }) => (
+    <div className={styles.field}>
+      <label className={styles.label}>
+        {label}
+      </label>
+      <input
+        {...input}
+        className={`${styles.input} ${touched && error && styles.inputError}`}
+        type={type}
+        placeholder={placeholder}
+      />
+      {touched && error && (<p className={styles.error}>{error}</p>)}
+    </div>
+  )
+
+  render () {
+    const { handleSubmit } = this.props;
+    return (
+      <div className={styles.login}>
+        <form className={styles.loginForm} onSubmit={handleSubmit}>
+          <Field
+            name="email"
+            type="email"
+            component={this.renderField}
+            label="Email"
+            placeholder="e.g. john.doe@email.com"
```

Comment:
Validation at field-level is something like this: https://redux-form.com/8.2.2/examples/fieldlevelvalidation/

Check that example, its a good way to reuse the validations.
ALSO in the case that you have many validations for just one field, create this validations in the component `constants.js` specifying the field name as key in a object, and the values must be an array containing the hierachy order of the validations something like:
```
import FORM_VALIDATIONS from '@constants/formValidations.js`

export const VALIDATIONS = {
email: [FORM_VALIDATIONS.required, FORM_VALIDATIONS.email],
...
};
```